Sources/Atomics/CMakeLists.txt

```diff
@@ -22,8 +22,12 @@ add_library(Atomics
   DoubleWord.swift)
 set_target_properties(Atomics PROPERTIES
   INTERFACE_INCLUDE_DIRECTORIES ${CMAKE_Swift_MODULE_DIRECTORY})
-target_compile_definitions(Atomics PRIVATE
+target_compile_definitions(Atomics PUBLIC
   ENABLE_DOUBLEWIDE_ATOMICS)
+if(CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|AMD64")
+  target_compile_options(Atomics PUBLIC
```

Comment:
Similar (anyone using Atomics should add `-Xcc -mcx16`?

Reply:
I think so! Atomic operations are all inlined into client code; and the 16-byte intrinsics should be generated as direct double wide instructions, they should never fall back to locking.

(Ideally though we should fix the compiler to force-enable cx16 on Linux/x86_64 -- IIRC the runtime is already built with this anyway, so it's not like we support running on CPUs that don't have it.)

Reply:
Yes, the runtime already requires cx16. We should upgrade SwiftPM to have the same baseline as the runtime.